modin/engines/ray/cudf_on_ray/frame/data.py

```diff
@@ -11,6 +11,8 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""Module houses implementation of ``PandasOnRayFrame`` class using cuDF."""
```

Comment:
"""Module houses class that implements ``PandasOnRayFrame`` class using cuDF."""

Reply:
fixed